src/status_im2/contexts/chat/messages/list/style.cljs

```diff
@@ -0,0 +1,83 @@
+(ns status-im2.contexts.chat.messages.list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]
+            [react-native.reanimated :as reanimated]))
+
+(defonce ^:const cover-height 168)
+(defonce ^:const overscroll-cover-height 2000)
+(defonce ^:const cover-bg-color "#2A799B33")
+(defonce ^:const header-avatar-top-offset -36)
+
+(def footer
+  {:z-index 5})
+
+(defn keyboard-avoiding-container
+  [view-height keyboard-height]
+  {:position :absolute
+   :display  :flex
+   :flex     1
+   :top      0
+   :left     0
+   :height   (- view-height keyboard-height)
+   :right    0})
+
+(def list-container
+  {:padding-top    16
+   :padding-bottom 16})
+
+(defn header-container
+  [insets]
+  {:scale-y          (if platform/android? -1 0)
+   :background-color (colors/theme-colors colors/white colors/neutral-95)
+   :top              (if platform/ios?
+                       (- 0 overscroll-cover-height)
+                       (:top insets))
+   :margin-bottom    (if platform/ios?
+                       (- 0 overscroll-cover-height)
+                       0)})
+
+(defn header-cover
+  [cover-bg-color status-bar-height]
+  {:width            "100%"
```

Comment:
It may be possible to achieve the same results with `:flex 1`

https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#dont-use-percents-to-define-widthheight